strangerstudios/paid-memberships-pro
# Wisdom Tracking and Integration code along with Setting to toggle opt-in status

### All Submissions:






### Changes proposed in this Pull Request:



Includes the Wisdom Tracker and adds the ability to opt-in to stats collection that helps us build a better product and support customers with more insight.

### How to test the changes in this Pull Request:

1. To test this locally, you'll want to use `add_filter( 'wisdom_is_local_paid-memberships-pro', '__return_false', 15 );` to override the normal local detection (stats opt-in will not collect or show the notice if on local).
2. The new opt-in notice will show up only on the PMPro Settings pages, we can adjust this and expand to more PMPro screens but I felt this was a good balance.
3. Click Allow or Do Not Allow and the Wisdom Tracker will respect that selection.
4. You can go to Advanced Settings > Other Options to revisit your opt-in choice and adjust it.

![Screen Shot 2022-02-04 at 9 08 37 AM](https://user-images.githubusercontent.com/709662/152553611-4eda0911-179f-4676-bf7e-b92d8082a6af.png)

![Screen Shot 2022-02-04 at 9 09 18 AM](https://user-images.githubusercontent.com/709662/152553621-00177187-1db1-4153-b3c6-1b63891ff32d.png)

### Other information:




### Changelog entry

> Added an opt-in stats collection so we can get better insight on how people use Paid Memberships Pro.
